### ISSUE TITLE
Suggestion: Marking which instances use external dependancies/trackers

### ISSUE THREAD

**Issue by BlueCatYellowCat** [Chocobozzz/PeerTube#1990]
Background: Many users of PeerTube have objections to trackers and also Google services (concrete evidence can be found in https://github.com/Chocobozzz/PeerTube/issues/1721).

Issue:
Some PeerTube instances choose to include external dependencies, including 2 in the 15 most popular:
_(simply looking at NoScript results, no in-depth checking was done)_
- **hostyour.tv** : popcash.net, pvclouds.com, waust.at (popcash.net is certainly a potentially unwanted site)
- **peertube.video** : googlesyndication.com, googletagmanager.com (Google services are controversial)

While I believe they have a right to do this (this is FLOSS, of course!), I also believe it is important that potential users are made aware. Maybe it would be good to add warnings on sites like https://instances.joinpeertube.org/instances to list potentially unwanted features.

Workaround:
_For some sites, these trackers could be avoided by viewing videos using an alternative Peertube site which federates with the tracked site. However, you would of course need to first be able to discover which sites federate with it! (Is there a publicly available graph of Peertube instance federation? That would be a nice feature...)_

**Comment by BlueCatYellowCat** [comment]
Changed title to use the word 'services', because 'dependencies' is literally incorrect as you can block them without the site breaking.

**Comment by Chocobozzz** [comment]
We could add another point in https://github.com/Chocobozzz/PeerTube/issues/1509: "Do you use trackers/Google services/ads?

**Comment by Gymcap** [comment]
I agree with that, I'd like to be made immediately aware if the instance I'm looking at is running trackers.
